scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -434,42 +312,37 @@ def build_test_packages(change_ids, obs_linked_project, home_project,
     # 'gerrit_project': Package()
     packages = {}
 
+    # The target branch associated with the first change is used for
+    # all changes
+    branch = None
```

Comment:
Are all projects always branched? Otherwise there may be cases where a common tooling project (for example) may stay on master with other projects regardless of branch using it. (I suspect this isn't a case we need to support).

Reply:
Right now they are because they are release stream specific, and the RPM related repos will likely always work in this model... However ardana-dev-tools, if it survives, may or may not stick to this model, but we can deal with that when the time comes... ;-)